src/main/java/com/revature/InterviewEvaluationsApplication.java

```diff
@@ -2,8 +2,11 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
+import com.revature.repositories.PersonRepository;
 
+@EnableJpaRepositories(basePackageClasses=PersonRepository.class)
 @SpringBootApplication
 public class InterviewEvaluationsApplication {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revaturelabs.com/coding_rules#rule_key=squid%3AS1118)